### DISCUSSION TITLE
ReactiveUI.Maui incompatible latest versions with Maui ?

### DISCUSSION THREAD

**Discussion by MateuszLas421** [reactiveui/ReactiveUI#3518] [General]
I have a problem, I am trying to update ReactiveUI.Maui from version 18.4.1 to 18.4.26. The new version has dependencies on Microsoft.WindowsAppSDK 1.2.230217.4. Only I guess this is the version that is incompatible with Maui (I get a bunch of errors when trying to install this version of WindowsAppSDK) ? Maui for even version 8.0.0-preview.1.7762 recommends Windows App SDK 1.2.221109.1  

`Detected package downgrade: Microsoft.WindowsAppSDK from 1.2.230217.4 to 1.2.221209.1. Reference the package directly from the project to select a different version.  
 xyz-> ReactiveUI.Maui 18.4.26 -> Microsoft.WindowsAppSDK (>= 1.2.230217.4) 
 xyz-> Microsoft.WindowsAppSDK (>= 1.2.221209.1)
`
Is this an error ? Do I need to troubleshoot in MAUI and configuration under Microsoft.WindowsAppSDK from 1.2.230217.4 ?

**Comment by glennawatson** [comment]
You got two projects (likely nuget)  wanting the windows sdk. 

Rxui wants newer 

One way that can work is referencing the newer version in your project. 

Should work without issue
